start.sh

```diff
@@ -1,2 +1,2 @@
 docker build . -t orchard
-docker run -it --rm -p 8888:8888 -v "${PWD}":/home/jovyan/work orchard
\ No newline at end of file
+docker run -it --rm -p 8888:8888 -e GRANT_SUDO=yes -v "${PWD}":/home/jovyan/work orchard
```

Comment:
Sorry. We committed this change by accident. Can you please remove the `-e GRANT_SUDO=yes` part? It is saver if the terminal in the notebook does not allow root access as depending how the container is launched, some parts of the host system might be exposed.